Source/HTTPHeaders.swift

```diff
@@ -368,12 +368,20 @@ extension HTTPHeader {
     /// Example: `iOS Example/1.0 (org.alamofire.iOS-Example; build:1; iOS 13.0.0) Alamofire/5.0.0`
     public static let defaultUserAgent: HTTPHeader = {
         let info = Bundle.main.infoDictionary
+        
+        #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
         let executable = (info?[kCFBundleExecutableKey as String] as? String) ??
             (ProcessInfo.processInfo.arguments.first?.split(separator: "/").last.map(String.init)) ??
             "Unknown"
         let bundle = info?[kCFBundleIdentifierKey as String] as? String ?? "Unknown"
         let appVersion = info?["CFBundleShortVersionString"] as? String ?? "Unknown"
         let appBuild = info?[kCFBundleVersionKey as String] as? String ?? "Unknown"
+        #else
+        let executable = "Unknown"
```

Comment:
Which of the constants aren't available? We could just make them literals.

Reply:
`kCFBundleExecutableKey`, `kCFBundleIdentifierKey`, and `kCFBundleVersionKey`, but I think it’s fairly unlikely that a Linux app would have those in an `infoDictionary`.

Reply:
Sure, but they can be added, so I think it would be good to have minimal Linux-specific behavior here and just use the fallbacks.

Reply:
These are now just literals. I think that it’s pretty unlikely they’ll change.